tests/utils.go

```diff
@@ -955,6 +943,71 @@ func NewRandomVMIWithEphemeralPVC(claimName string) *v1.VirtualMachineInstance {
 	return vmi
 }
 
+func NewRandomWindowsVMI() *v1.VirtualMachineInstance {
```

Comment:
Thanks! Instead of `NewRandowWindowsVMI` under `tests/utils` I opted for a `NewWindowsVMI` under `libvmi/factory`.
Regarding the move of most/all the utility functions from `utils` to `libvmi` I have some ideas, but I think this is out of the scope of this PR :) Thanks